### PR TITLE
add fallback state to GOB dashboard

### PR DESCRIPTION
**Pull request by TomWoodward**

### CODE REVIEW

File: cnxpublishing/views/admin.py [hunk 1]
```diff
@@ -516,7 +517,11 @@ def get_baking_statuses_sql(get_request):
                        f.sha1 as recipe,
                        m.module_ident,
                        ident_hash(m.uuid, m.major_version, m.minor_version),
-                       bpsa.created, ctm.status as state, ctm.traceback
+                       bpsa.created, ctm.traceback,
+                       CASE WHEN ctm.status = 'SUCCESS' and m.stateid = 8
```
Comment (pumazi) [inline cnxpublishing/views/admin.py]: add a `-- state = fallback` to the end of this line just to make it clear what `stateid = 8` means.

Comment (pumazi) [reply cnxpublishing/views/admin.py]: Or as you pointed out in chat... join modulestates and match on the name.

**Comment by pumazi** [comment]
Looking good so far. @TomWoodward, can you add a test for this change? I wouldn't think there is one already, but it's worth looking to see if something comes close.

**Comment by TomWoodward** [comment]
tried to add a test for the new state, but getting any of the test data into a state that isn't "pending" turned out to be non-trivial. there are no other tests for states other than pending, @pumazi has initiated "getting on with our lives" protocol.

**Comment by reedstrm** [comment]
Actually, there's code in test_subscribers that simulates a fallback. Could that be copied and used for GOB tests?

**Comment by reedstrm** [comment]
These tests pass locally. Not sure what to say. The test that fails (cnxpublishing/tests/views/test_publishing.py::BakeContentTestCase::test <- cnxpublishing/tests/testing.py FAILED ), which counts notification events,  is failing differently each time. 
Latest time it was 1 != 0. Previous run was 2 != 1

**Pull request merged**